frontend/src/components/Navigation/Navigation.css

```diff
@@ -0,0 +1,82 @@
+.nav {
+  max-width: 360px;
+  width: 100%;
+  position:relative
+
+}
+
+.nav__links {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  list-style: none;
+  font-family: 'Source Sans Pro', 'Franklin Gothic Medium', Arial, sans-serif;
+  padding: 0;
+}
+
+.nav__link {
+  text-decoration: none;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 18px;
+  line-height: 30px;
+  color: #000000;
+}
+
+.nav__link_active {
+  text-decoration: underline;
+}
+.nav__button {
+  display: none;
+  background-image: url(../../images/menu.png);
+  height: 48px;
+  width: 67px;
+  background-color: transparent;
+  background-repeat: no-repeat;
+  border: none;
+  cursor: pointer;
+  background-size: contain;
+  opacity: 1;
+  transition: opacity linear 0.2s;
+}
+
+.nav__button:hover{
+  opacity: 0.7;
+}
+
+@media screen and (max-width: 640px) {
+
+  .nav{
+    display: flex;
+    justify-content: end;
```

Comment:
I'd rather use 'flex-end' becasue 'end' alone has mixed support. Don't know if it'll make a difference after it's bundled though.

Reply:
Good catch. I'll fix it